core/src/mindustry/ui/fragments/PlacementFragment.java

```diff
@@ -271,9 +272,18 @@ public void build(Group parent){
                                     }
                                 }
                                 final String keyComboFinal = keyCombo;
+                                String modName = "";
+                                for(LoadedMod mod : mods.list()){
```

Comment:
You can use `modNameFinal = lastDisplay.minfo.mod != null ? lastDisplay.minfo.mod.meta.displayName : ""`

Reply:
thank you anuke, i had no idea blocks stored their mod info!
hell ill try and add it for placed blocks too, if i can find its fragment